File: src/main/java/com/mycompany/myapp/domain/enumeration/EtatDemande.java
package com.mycompany.myapp.domain.enumeration;

public enum EtatDemande {
    EN_ATTENTE, // Demande créée et en attente de validation DFC
    VALIDEE_DFC, // DFC a validé et renseigné le montantAccorde
    REFUSEE_DFC, // DFC a refusé
    EXECUTEE,
}

File: src/main/webapp/app/entities/enumerations/etat-demande.ts
export enum EtatDemande {
  EN_ATTENTE = 'EN_ATTENTE',

  VALIDEE_DFC = 'VALIDEE_DFC',

  REFUSEE_DFC = 'REFUSEE_DFC',

  EXECUTEE = 'EXECUTEE',
}

export const EnumEtatDemandeLabels = {
  [EtatDemande.EN_ATTENTE]: 'En attente',
  [EtatDemande.VALIDEE_DFC]: 'Validée',
  [EtatDemande.REFUSEE_DFC]: 'Refusée',
  [EtatDemande.EXECUTEE]: 'Exécutée',
};
